src/styles.css

```
@@ -709,3 +709,38 @@ footer a:hover{text-decoration:underline}
   color: var(--orange-deep);
   background: var(--orange-pale);
 }
+
+/* --- THEME TOGGLE TRANSITIONS --- */
+html.theme-transition, 
+html.theme-transition * {
+  transition: background-color 0.4s ease, color 0.4s ease, border-color 0.4s ease, fill 0.4s ease, stroke 0.4s ease, box-shadow 0.4s ease;
+}
+
+#themeToggleBtn {
```

Stylelint (17.12.0)

[error] Expected id selector "`#themeToggleBtn`" to be kebab-case (selector-id-pattern)

(selector-id-pattern)